common/src/main/java/com/wynntils/screens/lists/OverlayList.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.lists;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.OverlayManager;
+import com.wynntils.mc.render.Texture;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.screens.OverlaySelectionScreen;
+import com.wynntils.screens.lists.entries.OverlayEntry;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import net.minecraft.client.gui.components.ContainerObjectSelectionList;
+
+public class OverlayList extends ContainerObjectSelectionList<OverlayEntry> {
+    private static final int ITEM_HEIGHT = 25;
+    private final Set<Overlay> overlays;
```

Comment:
I don't really like how the list presented to the user is based on an (unordered) set. We should sort the set according to some principle (overlay name alphabetically, if nothing else).